pypeit/spectrographs/shane_kast.py

```diff
@@ -323,6 +259,8 @@ def __init__(self):
 
         # TODO: NEED TO CHECK ORIENTATION OF DATASEC AND OSCANSEC ARE
```

Comment:
Have we successfully done that now?  Or should we leave this comment in?

Reply:
it worked, so I think we are good.

Removed this TODO